tensorboard/webapp/widgets/line_chart_v2/lib/worker.ts

```diff
@@ -18,10 +18,15 @@ limitations under the License.
  *
  * This module exists to conform to internal requirements.
  *
- * @param workerResourcePath URL pathanme to the JavaScript resource served by
- *   TensorBoard. TensorBoard disallows fetching JavaScript resources from a differnt
- *   origin.
+ * @param workerJsFilename Worker JavaScript resource file name. Exact resource path
+ *   depends on context. It can be relative or absolute depending on a app and its
+ *   configuration (internally, it is absolute; i.e., served at "/<resource_name.js>").
```

Comment:
Perhaps consider replacing "internally, it is absolute..." with a redirection "please see the internal version of this file for details", in case this comment goes out of sync with internal implementation?